testcontainers-dapr/pom.xml

```diff
@@ -0,0 +1,135 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.dapr</groupId>
+        <artifactId>dapr-sdk-parent</artifactId>
+        <version>1.12.0-SNAPSHOT</version>
+    </parent>
+
+	<artifactId>testcontainers-dapr</artifactId>
+	<name>testcontainers-dapr</name>
+	<description>Testcontainers Dapr Module</description>
+	<packaging>jar</packaging>
+
+	<properties>
+		<snakeyaml.version>2.0</snakeyaml.version>
+		<testcontainers.version>1.19.1</testcontainers.version>
+		<logback-classic.version>1.4.12</logback-classic.version>
+        <wiremock.version>3.0.1</wiremock.version>
+	</properties>
+
+	<dependencies>
+		<dependency>
+			<groupId>org.yaml</groupId>
+			<artifactId>snakeyaml</artifactId>
+			<version>${snakeyaml.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.testcontainers</groupId>
+			<artifactId>testcontainers</artifactId>
+			<version>${testcontainers.version}</version>
+		</dependency>
+		<dependency>
```

Comment:
@salaboy do we really need this dependency? As far as I can see the testcontainsers module uses just plain Java SDK.